interactive_shell_handler.c

```diff
@@ -0,0 +1,150 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <windows.h>
+#include <psapi.h>
+#include <tchar.h>
+#include "interactive_shell_handler.h"
+#include "utils.h"
+
+// forward declarations
+void perform_command(char *command);
+char** split_command(char *command);
+
+void handle_copy(char* src, char* dst) {
+    int exitCode = 0;
+
+    exitCode = copy_file(src, dst);
+    if (exitCode == 0) {
+        printf("copied source file to destination");
+    } else {
+        printf("failed to copy source file to destination");
+    }
+}
+
+void handle_move(char* src, char* dst) {
+    int exitCode = copy_file(src, dst);
```

Comment:
Implementing move by using copy and then delete is not very efficient.
Moving a file in a filesystem is a very simple operation (we practically only change its location on disk) but copying a file requiring us to copy the entirety of its content.
Surely there must be an OS API to allow moving a file 😉